tests/v2/validation/workloads/workload_test.go

```diff
@@ -277,6 +339,20 @@ func validateDeploymentUpgrade(w *WorkloadTestSuite, namespaceName string, appv1
 	require.Equal(w.T(), expectedContainerCount, countPods)
 }
 
+func validateDeploymentScale(w *WorkloadTestSuite, namespaceName string, scaleDeployment *appv1.Deployment, image string, expectedContainerCount int) {
```

Comment:
can we make it a method of WorkloadTestSuite instead of a stand alone function that takes w as a parameter by passing the struct as a reciever to the method validateDeploymentScale i.e:  
```
func (w *WorkloadTestSuite) validateDeploymentScale(namespaceName string, scaleDeployment *appv1.Deployment, image string, expectedContainerCount int) {}
```
. that way we do not have to send w always while calling validateDeploymentScale and will have direct access to its fields and methods.

Reply:
Yes, I can, it's a great ideia